@xen-orchestra/lite/src/components/pool/dashboard/ramUsage/VmsRamUsage.vue

```diff
@@ -1,13 +1,14 @@
 <template>
-  <UsageBar :data="statFetched ? data : undefined" :n-items="N_ITEMS">
-    <template #header>
-      <span>{{ $t("vms") }}</span>
-      <span>{{ $t("top-#", { n: N_ITEMS }) }}</span>
-    </template>
-  </UsageBar>
+  <UiCardTitle
+    secondary
```

Comment:
I'm not a fan of the name of this prop.
At the first pass of review i was thinking this prop would define the color of the title. `primary | secondary | ...`
I suggest `subtitle` instead? Seems more understandable to me.

Reply:
Yes, I prefer that too, thanks.